src/partials/footer.html

```diff
@@ -0,0 +1,29 @@
+<footer class="footer">
+  <div class="footer__container">
+    <div class="footer__logo">
+      <svg class="footer__icon">
+        <use href="./images/sprite.svg#icon-logo"></use>
+      </svg>
+      <div class="footer__tablet">
+        <div class="footer__logo--tel">
+          <svg class="footer__icon--orange" width="15px" height="12px">
+            <use href="./images/sprite.svg#icon-phone"></use>
+          </svg>
+          <a class="footer__tel" href="+38 (000) 000 00 00" type="tel">+38 (000) 000 00 00</a>
+        </div>
+        <div class="footer__img">
+          <img
+            scr="./images/instagram.jpg"
+            srcset="./images/instagramx2.jpg"
+            alt="socials"
```

Comment:
Mało opisowy alt. Dla osób "dosłuchujących" strony przydałoby się użyć dłuższego opisu, np.: "Instagram icon - link to instagram." Czy coś w ten deseń.